### DISCUSSION TITLE
Pair programming Resources

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/virtualcoffee.io#516] [General Resources]
Do you have a resource (blogpost, directions, video, etc.) that you'd like to recommend to the community about pair programming?

**Comment by BekahHW** [comment]
## PAIR PROGRAMMING LIVE CODING TOOLS
*From 1/25 Coffee Chat*
- VSCODE liveshare
- [Duckly](https://duckly.com/)
- [Tuple](https://tuple.app)
- zoom screen sharing

**Reply by staszewski** [reply]
If you are the InteliJ user than there is a tool called Code With Me. More on that here: https://www.jetbrains.com/help/idea/code-with-me.html

Plugin is available here: https://plugins.jetbrains.com/plugin/14896-code-with-me

At least it was, it looks it is not maintained anymore. There is alternative though: https://plugins.jetbrains.com/plugin/14225-codetogether

**Comment by BekahHW** [comment]
## PAIR PROGRAMMING LIVE COLLABORATION/WHITEBOARDING TOOLS
*From 1/25 Coffee Chat*

- [Excalidraw](https://excalidraw.com/)
- [Miro](https://miro.com/)
